modin/pandas/base.py

```diff
@@ -11,7 +11,34 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
-"""Implement DataFrame/Series public API as pandas does."""
+"""
+Implement DataFrame/Series public API as pandas does.
+
+Methods docstrings contain tables with supported parameters information.
+These tables are structed as follows: The first column contains the parameter name,
+the second, third and fourth column - flags describing particular properties of
+method parameters for a concrete execution supported by Modin (`PandasOnRay`, `PandasOnDask`
+or `OmniSci`), and the last one is for parameter notes. The flags stand for the following:
+
++-----------+-----------------------------------------------------------------------------------------------+
+| Flag      | Meaning                                                                                       |
++===========+===============================================================================================+
+| Harmful   | Usage of this parameter can be harmful for performance of your application. Usually this      |
```

Comment:
Let's establish a name for this flag. Related discussion - [link](https://github.com/modin-project/modin/pull/4286#discussion_r867487158).

Reply:
cc @RehanSD, @devin-petersohn, take a look please